src/bridge/bridge-unix-sockets.cpp

```diff
@@ -33,6 +33,7 @@
 #include <filesystem>
 
 #define TMP_DIR "/tmp"
+#define SLIMEVR_HOME_LOCAL_DIR ".local/share/slimevr"
```

Comment:
```suggestion
#define XDG_DATA_DIR_DEFAULT ".local/share"
#define SLIMEVR_DATA_DIR "slimevr"
```

Reply:
i feel like these should be constants rather than macros, but i don't care enough to change it right now.